aiohttp/formdata.py

```diff
@@ -35,6 +37,10 @@ def __init__(self, fields:
     def is_multipart(self) -> bool:
         return self._is_multipart
 
+    @property
+    def is_processed(self) -> bool:
```

Comment:
Please don't expose the property.
I doubt if it is useful for something else than raising an exception of the double usage.